tests/test_call_parentheses.rs

```diff
@@ -0,0 +1,244 @@
+use stylua_lib::{format_code, CallParenType, Config, OutputVerification};
+
+fn format(paren_type_str: &str, input: &str) -> String {
+    let paren_type = match paren_type_str {
+        "Always" => CallParenType::Always,
+        "NoTable" => CallParenType::NoTable,
+        "None" => CallParenType::None,
+        _ => CallParenType::Always,
+    };
```

Comment:
Rather than matching a string, could we just pass in the enum variant directly to the function? Gives us an immediate compiler warning and reduces the likelihood of typos etc.